compose/galaxy-web/Dockerfile

```diff
@@ -57,7 +66,7 @@ RUN rm -f /usr/bin/startup && \
     --extra-vars galaxy_extras_config_nginx_upload=False \
     --extra-vars nginx_welcome_location=$NGINX_WELCOME_LOCATION \
     --extra-vars nginx_welcome_path=$NGINX_WELCOME_PATH \
-    --extra-vars startup_sleeplock=True \
+    --extra-vars startup_embed=True \
     --extra-vars startup_export_user_files=False \
```

Comment:
This one has no effect anymore when we move the startup.sh script to this repo I think.
Instead, we need to make sure "STARTUP_EXPORT_USER_FILES" is unset, which it should be by deafult.